corpus/literals.txt

```diff
@@ -1,43 +1,58 @@
 ==========================
-Multi-line strings
+Simple strings
 ==========================
 
-val string = """
+val oneLineString = "I'm just on one line"
+
+val multiLineString = """
   a
-  b
-  c
+  $thisIsntInterpolated
+  ${thisEither}
 """
 
 ---
 
 (compilation_unit
+  (val_definition (identifier) (string))
   (val_definition (identifier) (string)))
 
 ==========================
 Interpolated strings
 ==========================
 
-val string1 = "a $b c ${d} e"
+val string1 = s"a $b ${c}"
 
-val string2 = "$a"
+val string2 = f"hi $name%s"
```

Comment:
One future thing we can enhance even further here is identifying the formatters as identifiers, but for now there are a ton of other things that are probably more important to tackle than fine graining this.